pyproject.toml

```diff
@@ -22,7 +16,7 @@ known_first_party = [
     "pytorch_lightning",
     "tests",
 ]
-profile = "black"
+profile = "google"
```

Comment:
We could change the profile to `"google"`, but to minimize the changes in the codebase, could we keep the `"black"` profile? Fyi, by switching to google, the diff will look like:
```
-from tests.base.models import ParityModuleMNIST, ParityModuleRNN
+from tests.base.models import ParityModuleMNIST
+from tests.base.models import ParityModuleRNN
```

Reply:
It'd be weird to use the google/yapf formatter without the google isort config.

If the issue is 1 rule, we can always override it.

Reply:
after full use of yapf we do not need isort anymore :]

Reply:
@Borda I couldn't find the feature in yapf docs, but does yapf sort imports, too? or we don't need sorting import statements anymore?